app/uid/models.py

```diff
@@ -21,6 +22,79 @@ def check_neo4j_connection():
             time.sleep(1)  # Wait before retrying
     return False
 
+#class Alias(StructuredNode):
+ #   alias = StringProperty(unique_index=True)  # The alias name
+  #  context = StringProperty(required=False, default=None)  # Optional context
+   # points_to = RelationshipTo('NeoTerm', 'POINTS_TO')
+
+    #def __str__(self):
+     #   return self.alias
+
+# Define the NeoTerm class
+#class NeoTerm(StructuredNode):
+ #   name = StringProperty(unique_index=True)  # The name of the term
+  #  definition = StringProperty()  # A definition of the term (optional)
+    
+   # def __str__(self):
+    #    return self.name
+
+#class Alias(StructuredNode):
+ #   alias = StringProperty(unique_index=True)  # The alias name
+  #  context = StringProperty(required=False, default=None)  # Optional context
+   # points_to = RelationshipTo('NeoTerm', 'POINTS_TO')  # The relationship to NeoTerm
+
+    #def __str__(self):
+     #   return self.alias
+
+    #def link_to_term(self, neo_term):
+     #   """Link this alias to a NeoTerm."""
+      #  if isinstance(neo_term, NeoTerm):
+       #     self.points_to.connect(neo_term)
+
+    #def save(self, *args, **kwargs):
+     #   """Override the save method to automatically link the alias to a NeoTerm if context is provided."""
+      #  super(Alias, self).save(*args, **kwargs)
+
+       # if self.context:  # If context is provided, find the NeoTerm by name and link it
+        #    term = NeoTerm.nodes.get_or_none(name=self.context)
+         #   if term:
+          #      self.link_to_term(term)
+
+
+class Alias(StructuredNode):
```

Comment:
This class looks like it's more related to terms yeah?  Why is it in the UID portion of the project etc.

Reply:
They are in UID folder as I asked Florian if he had an ideal location or a preference for the search function location and he said I could leave it in the UID folder.